node/node.go

```diff
@@ -215,7 +215,6 @@ type Node struct {
 	indexerService    *txindex.IndexerService
 	prometheusSrv     *http.Server
 
-	ipfsAPI   iface.CoreAPI
```

Comment:
removed this for now, as we weren't using it anywhere. We can easily add it back in if needed.